modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -59,9 +59,13 @@ Run performance and load testing on your Mule applications to determine the numb
 
 == Internal Load Balancer
 
-Inbound traffic is processed using an internal load balancer managed by Anypoint Runtime Fabric. Since this load balancer is responsible for TLS termination, the amount of resources required scales based on the number of incoming connections and the average payload size for each request.
+Inbound traffic is processed using an internal load balancer managed by Anypoint Runtime Fabric. Since this load balancer is responsible for TLS termination, the number of resources required scales based on the number of incoming connections and the average payload size for each request.
```

Comment:
Since > Because

Text throughout the rest of this topic set uses "amount of resources." It's a slippery slope, determining what is considered an "aggregate" of countable things, but I agree with your correction here. Please GLOBALLY search for "amount of resources" and change it to "number of resources."